components/ghost/new-author.js

```diff
@@ -0,0 +1,36 @@
+const ghost = require("https://github.com/PipedreamHQ/pipedream/components/ghost/ghost.app.js");
+
+module.exports = {
+  name: "New Author",
+  description: "Emits an event for each new author added on a site.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ghost,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+
+  async run(event) {
```

Comment:
It is maybe okay to just rely on the deduper, but it would be nice to use `this.db` to checkpoint and minimize api calls if the api allows?

Reply:
I updated the api call to use a filter to minimize results to authors with IDs greater than the highest ID we've previously received.